benchmarks/Benchmarks.Trace/AgentWriterBenchmark.cs

```diff
@@ -85,9 +81,9 @@ public void AddHeader(string name, string value)
 
             public async Task<IApiResponse> PostAsync(Span[][] traces, FormatterResolverWrapper formatterResolver)
             {
-                using (var requestStream = new MemoryStream())
+                using (var requestStream = new NullStream())
```

Comment:
Should we use [`System.IO.Stream.Null`](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.null) instead of writing our own? (level of care low)

```suggestion
                using (var requestStream = Stream.Null)
```

Reply:
TIL